src/parsers/babel_parser.js

```diff
@@ -10,28 +10,27 @@
  */
 
 import {readFileSync} from 'fs';
-
-import {File as BabylonFile, Node as BabylonNode, parse as babylonParse} from 'babylon';
+import {File as BabelFile, Node as BabelNode} from '@babel/types';
+import * as parser from '@babel/parser';
 import type {ParsedNodeType} from './parser_nodes';
 import {NamedBlock, ParsedRange, ParseResult, ParsedNode} from './parser_nodes';
 
-// eslint-disable no-underscore-dangle
-const _getASTfor = (file: string, data: ?string): [BabylonFile, string] => {
+const _getASTfor = (file: string, data: ?string, options: parser.ParserOptions): [BabelFile, string] => {
```

Comment:
Since we are using flow, shouldn't the it be `?parser.ParserOptions`? To me `options` seems to be optional...

Reply:
absolutely, after an optional param there cannot be a required param.